packages/bee-q/src/components/dropdown/__tests__/bq-dropdown.e2e.ts

```diff
@@ -0,0 +1,21 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('bq-dropdown', () => {
+  it('should render', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<bq-dropdown></bq-dropdown>');
```

Comment:
Please update all the e2e tests to use the shorthand for setting the page content:

```suggestion
    const page = await newE2EPage({
      html: `<bq-dropdown></bq-dropdown>`
    });
```